app/presenters/sample_presenter.rb

```diff
@@ -0,0 +1,56 @@
+class SamplePresenter
+  delegate_missing_to :@sample
+
+  def self.map(samples, format)
+    samples.map { |sample| new(sample, format) }
+  end
+
+  def initialize(sample, format)
+    @sample = sample
+    @format = format
+  end
+
+  [
+    :batch_number,
+    :concentration,
+    :concentration_formula,
+    :replicate,
+    :virus_lineage,
+  ].each do |attr_name|
+    define_method attr_name do
+      if blinded_attribute?(attr_name)
+        blinded_value
+      else
+        @sample.__send__(attr_name)
+      end
+    end
+  end
```

Comment:
I'd consider this list of affected attributes a smell. It's a duplicate that needs to be kept in sync with the attributes returned by `Box#blind_attributes`.
Two possible solutions:

* Add this wrapper for *all* delegated attributes. This means essentially replacing `delegate_missing_to` with a custom `method_missing`. This is practically already happening here for the affected attributes.
* Refactor `Box#blind_attributes` to a class method that returns all blindable attributes when no purpose is given and use this list here.